.github/workflows/main.yml

```diff
@@ -16,8 +16,8 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           browser: chrome
-          build: yarn run build
-          start: yarn start
+          build: next build
+          start: npm start
```

Comment:
Hey cotts, just to point at that we use yarn as the default package manager in most of our project

Reply:
ok, I will change that.

Reply:
done